frontend/src/components/SignUp.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+const SignUp = () => {
+  const handleSubmit = (evt) => {
+    evt.preventDefault();
+    console.log("Form submitted");
+  };
+
+  return (
+    <>
+      <form method="post" action="" onSubmit={handleSubmit}>
+        <fieldset>
+          <legend>Sign up for personalized wine recommendations</legend>
+
+          <label htmlFor="firstname">
+            First Name
+            <input type="text" id="firstname" name="firstname"></input>
+          </label>
```

Comment:
One last tip, if you wrap the `input` with the `label` tag, you don't need to use an `id` and point the `htmlFor` to the `id` of the `input`. The `label` will be already linked to the `input` and we don't need to manage ids this way.